install-yang-modules.sh

```diff
@@ -13,7 +13,10 @@ set -e
 # Skip submodules as they will automatically be installed with their parent
 for f in ${MODULES}; do
 	n=$((n+1))
-	if head -n1 $f | grep ^submodule > /dev/null 2>&1; then echo "[$n/$total] Skipping submodule $f"; continue; fi
```

Comment:
I have changed this because in some standard submodules the `submodule` keyword is not at the beginning of the first line